src/pairwise.jl

```diff
@@ -0,0 +1,88 @@
+_allocmatrix(X, Y, T) = Matrix{T}(undef, length(X), length(Y))
+_allocmatrix(X, T) = Matrix{T}(undef, length(X), length(X))
+
+import Distances: pairwise
+
+@doc """
+    pairwise(dist::StringDistance, itr; eltype = Float64, precalc = nothing)
```

Comment:
could you add a `nthreads = Threads.nthreads` option to allow users to change the number of threads?

Reply:
Ok, but I'm not sure how to control the number of threads used after Julia starts. We would have to manually spawn then or what do you propose?

Reply:
Right, nevermind. You cannot change the number of threads when using `@threads`.